docker/README.md

```diff
@@ -4,7 +4,7 @@
 
 >**Warning**
 >
->DO NOT use the dev configuration on a public network. The dev environment includes credentials to enable rapid development, and the MongoDB service config does not include authentication. This is not secure for public deployment.
+>DO NOT use the included Docker compose configuration on a public network. The environment and docker-compose.yml files contain credentials and do not enforce MongoDB authentication. This enables rapid, local-only development and is not secure for public deployment.
```

Comment:
The phrase, ' files contain credentials and do not enforce MongoDB authentication' is confusing. Please change to 'CVE Services credentials' or 'development credentials' to distinguish between that and credentials for MongoDB